src/components/Login/Login.tsx

```diff
@@ -7,37 +8,64 @@ import {
   Button,
   FormGroup,
   FormControlLabel,
+  Modal
 } from '@mui/material/';
 import CustomTextFieldGroup from '../CustomTextFieldGroup/CustomTextFieldGroup';
-import { LoginTextFields } from './LoginTextFieldsData';
+import { LoginTextFields, SignUpTextFields } from './AuthTextFieldsData';
 
-const Login = ({ onClose }: { onClose: () => void }) => {
+interface AuthModalProps {
```

Comment:
nit-pick: since this interface is not exported and is only used here, we can quickly assume that this is the Component props (this comment is non-blocking by nature).

```suggestion
interface Props {
```